app/qml/main.qml

```diff
@@ -119,6 +120,18 @@ ApplicationWindow {
         }
     }
 
+    function toggleFullScreen() {
+        onActivated: {
```

Comment:
What does onActivated mean here? The syntax looks odd; I'd remove it if it works without it.

Reply:
Shouldn't be there, left over code when I had it directly inside the toolbutton.

Reply:
Fixed.